packages/google_sign_in/google_sign_in_android/test/google_sign_in_android_test.dart

```diff
@@ -101,16 +101,18 @@ void main() {
   test('Other functions pass through arguments to the channel', () async {
     final Map<Function, Matcher> tests = <Function, Matcher>{
       () {
-        googleSignIn.init(
```

Comment:
Here and on iOS, please add a new test instead of changing the old one, so that we have coverage that `init` hasn't regressed (and doesn't later).